.gitignore

```diff
@@ -1,2 +1,4 @@
 composer.lock
-vendor
\ No newline at end of file
+vendor
+.idea
+*~
```

Comment:
Regarding the `*~` gitignore entry: are that backup files created by the Tiqr server, or files created by your IDE or other development specific 'thing'? 

If the first is the case then this rule is fine with me. If the files are create by an outside factor (IDE, ..) then please add that rule to you personal (global) gitignore.

Reply:
Thank you for the suggestion; I've changed this.